diesel/src/connection/mod.rs

```diff
@@ -4,12 +4,16 @@ use query_source::Queryable;
 use result::*;
 use types::HasSqlType;
 
+/// Trait to perform a simple connection with the associated backend.
 pub trait SimpleConnection {
     #[doc(hidden)]
     fn batch_execute(&self, query: &str) -> QueryResult<()>;
 }
 
+/// Adds `establish`, `transaction`, `test_transaction` and `begin_test_transaction`
+/// to all types that implements `SimpleConnection` trait.
```

Comment:
While technically right, this is the same information that rustdoc shows you :)

Mentioning that it extend tre SimpleConnection trait is a good idea though.

Maybe it helps to instead answer questions like "When do I use this?", or "Where does this fit in my application?", or "What problem does this solve?". Or, as this is a trait, "What does this abstract over?"